styles.css

```diff
@@ -0,0 +1,1263 @@
+  font-family: 'Lato', sans-serif;
+  line-height: 170%;
+  font-size: 3vw;
+  font-weight: 700;
+  text-align: center;
+  padding-bottom: 10%;
+}
+
+.mobile-menu a:visited {
+  text-decoration: none;
+  color: #e4ddc9;
+}
+
+.Hello {
+  background-color: rgb(247, 247, 247, 0.97);
+  background-image: url(resources/images/nunsplash.jpg);
+  background-repeat: no-repeat;
+  background-size: 100%;
+  background-blend-mode: overlay;
+  padding-top: 5%;
+}
+
+.hello {
+  font-family: "Lato", sans-serif;
+  color: #ec5242;
+  font-size: 5vw;
+  margin-left: 6%;
+  background-color: transparent;
+}
+
+.hello.about {
+  text-align: center;
+  margin-left: 25%;
+}
+
+#hello-about {
+  margin-left: 23%;
+}
+
+.summit-name {
+  color: rgba(300, 35, 12, 0.7);
+  font-family: Cocogoose, sans-serif;
+  font-size: 7.2vw;
+  margin-top: 0;
+  margin-left: 6%;
+  margin-right: 1%;
+  background: url(resources/images/unsplash.jpg);
+  -webkit-background-clip: text;
+  background-size: 50%;
+}
+
+.summit-name.about {
+  text-align: center;
+  margin-top: 4%;
+  margin-left: 6%;
+  margin-right: 6%;
+  color: rgba(236, 82, 66, 1);
+}
+
+.summit-description {
+  font-family: 'Lato', sans-serif;
+  border: solid 3px white;
+  margin-right: 5%;
+  margin-left: 5%;
+  padding: 5%;
+  line-height: 170%;
+  font-size: 3vw;
+}
+
+.date-location {
+  padding-bottom: 12%;
+  background-color: transparent;
+  margin-left: 6%;
+}
+
+.date {
+  font-family: Cocogoose, sans-serif;
+  color: #5d5d5d;
+  font-size: 5.5vw;
+  margin-bottom: 3%;
+  background-color: transparent;
+}
+
+.location {
+  font-family: 'Lato', sans-serif;
+  color: #5d5d5d;
+  font-size: 3.5vw;
+  margin-top: 0;
+  background-color: transparent;
+}
+
+.main-program {
+  display: flex;
+  flex-direction: column;
+  justify-items: center;
+  background-color: #272a31;
+  margin-left: 0;
+  background-image: url(resources/images/unsplash.jpg);
+  background-repeat: no-repeat;
+  background-size: 180%;
+  background-blend-mode: multiply;
+}
+
+.main-program h2 {
+  background-color: transparent;
+  color: white;
+  font-family: 'Lato', sans-serif;
+  text-align: center;
+  font-size: 5vw;
+  position: relative;
+  padding-bottom: 3%;
+}
+
+.main-program h2::before {
+  content: "";
+  position: absolute;
+  width: 10%;
+  height: 1px;
+  bottom: 0;
+  left: 45%;
+  border-bottom: 1px solid #ec5242;
+}
+
+.events {
+  margin-left: 3%;
+  margin-right: 3%;
+  padding-left: 0;
+  background-color: transparent;
+}
+
+.event-card {
+  display: flex;
+  background-color: rgb(93, 93, 93, 0.5);
+  color: white;
+  list-style: none;
+  font-family: 'Lato', sans-serif;
+  margin-bottom: 2%;
+  border: solid 2px rgb(93, 93, 93, 0.1);
+  min-height: 200px;
+}
+
+.event-card:hover {
+  border: solid 2px white;
+}
+
+.event-type {
+  background-color: transparent;
+  font-family: 'Lato', sans-serif;
+  font-weight: 900;
+  color: #ec5242;
+  font-size: 4vw;
+  align-self: center;
+  width: 60px;
+  margin-left: 5px;
+}
```

Comment:
- [OPTIONAL] Please, be careful when you are implementing changes in your code. Sometimes It generates changes in others parts of your project as the alignment of your text in this case. (Following the guidelines, I marked this comment as an OPTIONAL because you implemented these changes after your first review but before mine. I kindly suggest fixing it. It will help you to have a great project to show in your portfolio.) 😉 

![capstone-cards-aligment](https://user-images.githubusercontent.com/79876959/142704677-b65366a3-202e-495f-84b3-574c68067f49.PNG)